### PR TITLE
Policy of Merge Pull Request

### PR DESCRIPTION
**Pull request by kito-cheng**
This document is about the policy of how to proposal new stuffs for psABI, and describe the merge/accept criteria for different kind of changes.

Initial version was proposed in the psABI meeting, and we have several revision after got feedback from the community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,6 @@ RISC-V Assembly Programmer's Manual | https://github.com/riscv/riscv-asm-manual
 RISC-V Toolchain Conventions        | https://github.com/riscv/riscv-toolchain-conventions  | RISC-V-specific toolchain behavior and command line option.
 RISC-V Semihosting Spec             | https://github.com/riscv/riscv-semihosting-spec       | Spec for semihosting.
 RISC-V Supervisor Binary Interface  | https://github.com/riscv/riscv-sbi-doc                | Spec for SBI.
+
+# Links
+[Policy of Merge Pull Request](policy.md)
```
Comment (asb) [inline README.md:25]: "Policy for Merging Pull Requests"?

File: policy.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Policy for Merging Pull Request
```
Comment (asb) [inline policy.md:1]: Request => Requests?

**Review by asb — COMMENTED** [review]
I've left two tiny typographical suggestions, but for what it's worth I like this policy.

File: README.md (hunk 1 above)
Comment (kito-cheng) [reply README.md]: Updated :)

File: policy.md (hunk 2 above)
Comment (kito-cheng) [reply policy.md]: Thanks, updated.

**Comment by kito-cheng** [comment]
This PR has been asked RISC-V LLVM community and RISC-V GNU community, also discussed on toolchain & runtime TG and psABI TG, and wait enough review review period, so I gonna merge this PR, but if anyone has any suggestion are welcome a PR or an issue!

**Pull request merged**